debian/CellServDB.athena

```diff
@@ -1,7 +1,7 @@
->athena.mit.edu         #MIT/Athena cell
+>athena.mit.edu         #Cell Name
```

Comment:
... or, at least, that's what CellServDB.new had last night, it was apparently fixed in the live CellServDB. Updated to match in 1c542c4 so that it builds.